web/srv/server.go

```diff
@@ -173,6 +173,7 @@ func NewServer(
 	server.router.GET("/api/routes", handler.handleAPITopRoutes)
 	server.router.GET("/api/edges", handler.handleAPIEdges)
 	server.router.GET("/api/check", handler.handleAPICheck)
+	server.router.GET("/api/namespaces/:namespace/:resource-type/:resource-name", handler.handleAPIResourceDefinition)
```

Comment:
We probably should change this URL scheme so it is consistent with the other API paths. So it would look something like this:
```diff
diff --git a/web/srv/server.go b/web/srv/server.go
index 9f0c05b1..dff692ff 100644
--- a/web/srv/server.go
+++ b/web/srv/server.go
@@ -173,7 +173,7 @@ func NewServer(
 	server.router.GET("/api/routes", handler.handleAPITopRoutes)
 	server.router.GET("/api/edges", handler.handleAPIEdges)
 	server.router.GET("/api/check", handler.handleAPICheck)
-	server.router.GET("/api/namespaces/:namespace/:resource-type/:resource-name", handler.handleAPIResourceDefinition)
+	server.router.GET("/api/resource-definitions", handler.handleAPIResourceDefinition)
 
 	// grafana proxy
 	server.router.DELETE("/grafana/*grafanapath", handler.handleGrafana)
```

This would mean we would use query parameters to pass resource information to the handler. See full diff above.